extensions/positron-catalog-explorer/src/catalogs/databricks.ts

```diff
@@ -113,6 +114,24 @@ async function registerDatabricksCatalog(
 	if (!token) {
 		return;
 	}
+
+	// Validate the token by checking if the user is available
+	try {
+		const userResponse = await fetch(`${workspace}/api/2.0/workspace/list?path=/&limit=1`, {
```

Comment:
Are there meant to be 2 toasts? (this is fine if so)

<img width="459" height="232" alt="Image" src="https://github.com/user-attachments/assets/31bbc289-6f1c-4520-8aa3-6d635f7775b1" />